src/components/TestimonialsSection/TestimonialsSection.jsx

```diff
@@ -70,26 +68,37 @@ const TestimonialsBox = styled.div`
 `;
 
 const TestimonialsSection = (props) => {
-  const Emily = new User(`Emily R.`, `Marketing Director`, userEmilyImg);
-  const Thomas = new User(`Thomas S.`, `Chief Operating Officer`, userJennyImg);
-  const Jennie = new User(`Jennie F.`, `Business Owner`, userThomasImg);
+  const usersObj = {};
+  const Emily = new User(

```

Comment:
class instances should be names using `thisNotation`, `ThisNotation` is reserved for classes only. So

`const emily = new User(....)`